code/game/objects/items/grenades/atmos_grenades.dm

```diff
@@ -23,53 +23,70 @@
 		SEND_SIGNAL(src, COMSIG_MOB_GRENADE_ARMED, user, src, det_time, delayoverride)
 	addtimer(CALLBACK(src, .proc/detonate), isnull(delayoverride)? det_time : delayoverride)
 
-/obj/item/grenade/gas_crystal/healium_crystal
-	name = "Healium crystal"
-	desc = "A crystal made from the Healium gas, it's cold to the touch."
+/obj/item/grenade/gas_crystal/cleansing_crystal
+	name = "cleansing crystal"
+	desc = "A crystal made from various gasses condensed down into a solid, when detonated it will scrubs out all the gasses aside from oxygen and nitrogen and refill within a 5 by 10 area"
 	icon_state = "healium_crystal"
 	///Range of the grenade that will cool down and affect mobs
-	var/fix_range = 7
+	var/fix_range = 50
```

Comment:
don't call it range, call it turf_count, or fixed_count or something like that. also update the comment to better describe behavior